index.html

```diff
@@ -2356,9 +2401,39 @@
       iframe.addEventListener('destroy', () => {
         screenQuad.material.uniforms.numTextures.value = 1;
       });
-      container.appendChild(iframe);
-
-      iframes.push(iframe);
+        // function to test the URL on submit for a 404
+        function checkURL() {
+            let request;
+            if(window.XMLHttpRequest)
+                request = new XMLHttpRequest();
+            else
+                request = new ActiveXObject("Microsoft.XMLHTTP");
+                try{
+                    console.log("Loading");
+                    request.open('GET', u, false);
+                    request.send(); // there will be a 'pause' here until the response to come.
+                }
+                catch{
+                    console.log("URL failed to load");
+                    return false;
+                }
+            return true;
+        }
+        // if URL 404, throw error message
+        if(checkURL(u) === false){
+            loadingBarState.ifLoading = true;
+            loadingBarState.ifError = true;
+            menuMesh.urlMesh.updateText();
+        }
+        // if URL is correct and responds, show "loaded mesh" to user
+        else if(checkURL(u) === true){
```

Comment:
If my understanding is correct, when we call this again here it would create _another_ request for the success case, so therefore the solution potentially triples the reality tab load time -- even when it works!

😓